### DISCUSSION TITLE
Map: 65 devices and it's a disaster!

### DISCUSSION THREAD

**Discussion by andreapx** [Koenkk/zigbee2mqtt#19228] [General]
Hi!
I've 65 devices and when I create the map in the web UI, it throws out something that can't be understood, like this:
![09-10-2023_10-55-54_9_Zigbee2MQTT_—_Mozilla_Firefox](https://github.com/Koenkk/zigbee2mqtt/assets/13846728/c3d8099f-dc27-4ffd-88f7-519641bdd791)
How can I create a clearer map?
Thanks!

**Comment by BullionMan** [comment]
I've got a messy map these days too! I have some other weirdness lately that seems to be coming from Z2M. Two automation in Home Assistant are causing Ikea bulbs to not come on at 100%, but at something like 10%, although it shows 100% but it isn't.
![image](https://github.com/Koenkk/zigbee2mqtt/assets/24967571/0de2a5f1-2f88-481d-a42e-17941039f469)
